src/Passable.js

```diff
@@ -65,48 +65,60 @@ class Passable {
 
         this.res.initFieldCounters(fieldName);
 
-        if (typeof test !== 'function' && !(test instanceof Promise)) {
-            return;
+        if (typeof test === 'function') {
+            this.runTest(Object.assign(test, {
```

Comment:
You're merging the objects anyhow, so how about:
```js
let operation;

if (typeof test === 'function') {
    operation = 'runTest';
} else if (test instanceof Promise) {
    operation = 'addPendingTest';
} else {
    return;
}

Object.assign(test, {
    fieldName,
    statement,
    severity
});

this[operation](test);
```

Reply:
Nice!